app/controllers/api/v1/comments_controller.rb

```diff
@@ -0,0 +1,29 @@
+class Api::V1::CommentsController < ApplicationController
+  before_action :fetch_post
+
+  def index
+    @comments = @post.comments
+    render json: @comments
+  end
+
+  def create
+    @comment = current_user.comments.build(comment_params)
+    @comment.post = @post
+    if @comment.save
+      render json: @comment, status: :created
+    else
+      render json: { error: 'Failed to create comment' }, status: :unprocessable_entity
+    end
+  end
```

Comment:
- [ ] It would be better if we can create a comment in Postman as we did the 2 endpoints successfully as seen in the screenshot below. The post request for creating a comment on the endpoint  `api/v1/users/:user_id/posts/:id/comments` returns `Can't verify CSRF token authenticity` as seen below
![Screenshot from 2023-05-05 11-52-07](https://user-images.githubusercontent.com/46388113/236450281-02592564-dfe7-478a-b958-671ec2db0cdf.png)
![Screenshot from 2023-05-05 11-57-13](https://user-images.githubusercontent.com/46388113/236450521-df5c90d4-4859-4957-b860-432645825176.png)

Reply:
@addod19 , I have fixed it
![Screenshot from 2023-05-05 14-19-19](https://user-images.githubusercontent.com/110976050/236468968-25b4cb33-4c3c-45a8-b235-1814920024d5.png)